lib/galaxy/authnz/psa_authnz.py

```diff
@@ -42,6 +42,7 @@
     "elixir": "social_core.backends.elixir.ElixirOpenIdConnect",
     "okta": "social_core.backends.okta_openidconnect.OktaOpenIdConnect",
     "azure": "social_core.backends.azuread_tenant.AzureADV2TenantOAuth2",
+    "checkin": "social_core.backends.checkin.CheckinOpenIdConnect",
```

Comment:
Would it be ok to name this egi_checkin? I have the feeling that `checkin` is maybe a bit too generic. But maybe its just me?

Reply:
indeed. Probably I should also change the psa PR, no?

Reply:
That I don't know :)
All I can say is that I was searching for "EGI" in PSA and did not find it - so I ask last week Sebastian if that would be a nice idea to add - I just learned today that it is called Checkin and that there is a PR :)

Reply:
this is fixed now